### PR TITLE
Add French configure-pod-configmap

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -0,0 +1,693 @@
...
+
+1. Définissez une variable d'environnement comme paire clé-valeur dans un ConfigMap:
+
+    ```shell
+    kubectl create configmap special-config --from-literal=special.how=very
+    ```
+
+1. Attribuez la valeur `special.how` défini dans ConfigMap à la variable d'environnement `SPECIAL_LEVEL_KEY` dans la spécification du Pod.
+
+   {{< codenew file="pods/pod-single-configmap-env-variable.yaml" >}}
```
Comment (rekcah78) [inline content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md:459]: the path is specific to the localization, you must copy the file to the fr branch,
/opt/build/repo/content/fr/examples/pods/pod-single-configmap-env-variable.yaml: no such file or directory

File: content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+very
+```
+
+{{< caution >}}
+Comme avant, tous les fichiers précédents dans le répertoire `/etc/config/` seront supprimés.
+{{< /caution >}}
+
+### Clés de projet pour des chemins et des autorisations de fichiers spécifiques
+
+You can project keys to specific paths and specific permissions on a per-file basis.
```
Comment (rekcah78) [inline content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md:607]: not translated

**Review by rekcah78 — CHANGES_REQUESTED** [review]

**Comment by rekcah78** [comment]
@remyleone https://deploy-preview-18962--kubernetes-io-master-staging.netlify.app/fr/docs/tasks/configure-pod-container/configure-pod-configmap/
seems to have a rendering problem

**Comment by remyleone** [comment]
@rekcah78 capture are not used anymore, therefore I needed to remove them :)

File: content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 3]
```diff
@@ -0,0 +1,693 @@
+---
+title: Configure a Pod to Use a ConfigMap
```
Comment (rekcah78) [inline content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md:2]: not translated

**Review by rekcah78 — CHANGES_REQUESTED** [review]

File: content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 4]
```diff
@@ -0,0 +1,684 @@
...
+
+La ressource API ConfigMap stocke les données de configuration sous forme de paires clé-valeur.
+Les données peuvent être consommées dans des pods ou fournir les configurations des composants du système tels que les contrôleurs.
+ConfigMap est similaire à [Secrets](/docs/concepts/configuration/secret/), mais fournit un moyen de travailler avec des chaînes de caractères qui ne contiennent pas d'informations sensibles.
+Les utilisateurs comme les composants du système peuvent stocker des données de configuration dans un ConfigMap.
+
+{{< note >}}
+Les ConfigMaps doivent faire référence aux fichiers de propriétés, et non les remplacer.
+Pensez à la ConfigMap comme représentant quelque chose de similaire au répertoire `/etc` de Linux et à son contenu.
+Par exemple, si vous créez un [volume Kubernetes] (/docs/concepts/storage/volumes/) à partir d'une ConfigMap, chaque élément de données de la ConfigMap est représenté par un fichier individuel dans le volume.
```
Comment (AWKIF) [inline content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md:631]: remove space
[volume Kubernetes] (/docs/concepts/storage/volumes/)

**Review by AWKIF — CHANGES_REQUESTED** [review]

File: content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md (hunk 3 above)
Comment (remyleone) [reply content/fr/docs/tasks/configure-pod-container/configure-pod-configmap.md]: Fixed :)

**Pull request merged**